Documentation/Index.rst

```diff
@@ -15,146 +15,115 @@ installed.
 
 ----
 
-.. container:: row m-0 p-0
+..  card-grid::
+    :columns: 1
+    :columns-md: 2
+    :gap: 4
+    :class: pb-4
+    :card-height: 100
 
-   .. container:: col-md-6 pl-0 pr-3 py-3 m-0
+    ..  card:: Concepts
 
-      .. container:: card px-0 h-100
+        Written for new users, this chapter introduces some of TYPO3's core
+        concepts including the backend, TYPO3's administration interface.
 
-         .. rst-class:: card-header h3
+        ..  card-footer:: :ref:`Learn about the basic concepts <Concepts>`
+            :button-style: btn btn-secondary stretched-link
 
-            .. rubric:: :ref:`Concepts <Concepts>`
+    ..  card:: System requirements
 
-         .. container:: card-body
+        System requirements for the host operation system, including its web
+        server and database and how they should be configured prior to
+        installation.
 
-            Written for new users, this chapter introduces some of TYPO3s core concepts including the backend, TYPO3s administration interface.
+        ..  card-footer:: :ref:`Go to the System Requirements <System-Requirements>`
```

Comment:
```suggestion
        ..  card-footer:: :ref:`Inspect the System requirements <System-Requirements>`
```

"System requirements" is lowercased everywhere else. Also "Go to" is odd, when regarding the other button labels, and I try to not duplicate "Learn about" (or "Investigate").